crates/electrum/tests/test_electrum.rs

```diff
@@ -22,10 +23,34 @@ fn get_balance(
     Ok(balance)
 }
 
+fn electrum_sync<I>(
+    client: &BdkElectrumClient<Client>,
+    spks: &[ScriptBuf],
+    batch_size: usize,
```

Comment:
I think it's okay to hard-code the batch size and have it as a constant at the top of the file (instead of having it as an input to this function). The value is not relevant for what we are testing and not having it slightly increases readability for the tests.